client/extension.ts

```diff
@@ -233,3 +244,20 @@ export function deactivate(context: vscode.ExtensionContext): void {
 function handleUnload(context: vscode.ExtensionContext, arrayOfUnloadableObject: vscode.Disposable[]) {
     arrayOfUnloadableObject.forEach(unloadableObject => context.subscriptions.push(unloadableObject as vscode.Disposable));
 }
+
+function newMarklogicTdeValidateClient(cfg: vscode.WorkspaceConfiguration): ClientContext {
+    const configParams: Record<string, unknown> = {
```

Comment:
One refactoring idea - I think this is the 4th place where this code exists - i.e. a Record being created based on WorkspaceConfiguration properties. Could this become a new constructor in MlClientParameters so that the various property names don't need to be repeated?

I'm specifically thinking for when `cloudApiKey` is added - the fewer places that need to be modified to support that, the better.